src/jdbc/cubrid/jdbc/driver/CUBRIDPreparedStatement.java

```diff
@@ -420,7 +420,9 @@ public synchronized void setObject(int parameterIndex, Object x, int targetSqlTy
                 } catch (Exception e) {
                     u_stmt.bind(parameterIndex - 1, x);
                 }
-                u_stmt.bind(parameterIndex - 1, new BigDecimal(n.toString()).setScale(scale));
+                if (n != null) {
+                    u_stmt.bind(parameterIndex - 1, new BigDecimal(n.toString()).setScale(scale));
+                }
             } else {
```

Comment:
Check indent.

Reply:
I already checked it with google-java-format program. Re-check, please.

Reply:
You right! I confused the **_if_** that matches the **_else_**.

Reply:
Same for me. it 's ok.

Reply:
Thank you for re-checking.